src/main/java/be/seeseemelk/mockbukkit/entity/FishHookMock.java

```diff
@@ -66,6 +72,79 @@ public void setMaxWaitTime(int maxWaitTime)
 		this.maxWaitTime = maxWaitTime;
 	}
 
+	@Override
+	public void setWaitTime(int min, int max)
+	{
+		this.setMinWaitTime(min);
+		this.setMaxWaitTime(max);
+	}
+
+	@Override
+	public int getMinLureTime()
+	{
+		return this.minLureTime;
+	}
+
+	@Override
+	public void setMinLureTime(int minLureTime)
+	{
+		Preconditions.checkArgument(minLureTime >= 0 && minLureTime <= this.getMaxLureTime(), "The minimum lure time should be between 0 and the maximum lure time.");
+		this.minLureTime = minLureTime;
+	}
+
+	@Override
+	public int getMaxLureTime()
+	{
+		return this.maxLureTime;
+	}
+
+	@Override
+	public void setMaxLureTime(int maxLureTime)
+	{
+		Preconditions.checkArgument(maxLureTime >= 0 && maxLureTime >= this.getMinLureTime(), "The maximum lure time should be higher than or equal to 0 and the minimum lure time.");
+		this.maxLureTime = maxLureTime;
+	}
+
+	@Override
+	public void setLureTime(int min, int max)
+	{
+		this.setMinLureTime(min);
+		this.setMaxLureTime(max);
+	}
+
+	@Override
+	public float getMinLureAngle()
+	{
+		return this.minLureAngle;
+	}
+
+	@Override
+	public void setMinLureAngle(float minLureAngle)
+	{
+		Preconditions.checkArgument(minLureAngle >= 0 && minLureAngle <= this.getMaxLureAngle(), "The minimum lure angle should be between 0 and the maximum lure angle.");
+		this.minLureAngle = minLureAngle;
+	}
+
+	@Override
+	public float getMaxLureAngle()
+	{
+		return this.maxLureAngle;
+	}
+
+	@Override
+	public void setMaxLureAngle(float maxLureAngle)
+	{
+		Preconditions.checkArgument(maxLureAngle >= 0 && maxLureAngle >= this.getMinLureAngle() && maxLureAngle <= 360, "The maximum lure angle should be higher than or equal to 0 and the minimum lure angle but also less than or equal to 360.");
```

Comment:
Can you please add a Linebreak. Having Lines this long makes editing and understanding the Code in the future harder